osg-pilot-container/osgpilot_meter

```diff
@@ -579,6 +584,75 @@ def main():
     send_updates(opts)
 
 
+def send_alternate_records(gratia_info, gratia_config):
+    """
+    For any accumulated records with an alternate probe/site name, send in a sub-process.
+    """
+    if not gratia_info:
+        return
+    GratiaCore.Disconnect()
+    for info, records in gratia_info.items():
+        pid = os.fork()
+        if pid == 0: # I am the child
+            try:
+                signal.alarm(5*60)
+                send_alternate_records_child(info, records, gratia_config)
+            except Exception as e:
+                DebugPrint(2, "Failed to send alternate records: %s" % str(e))
+                DebugPrintTraceback(2)
+                os._exit(0)
+            os._exit(0)
+        else: # I am parent
+            try:
+                os.waitpid(pid, 0)
+            except:
+                raise
```

Comment:
Why does this need to be in a try/except if we're just raising?

Reply:
Yeah it looks like the try/except there does not accomplish anything.

So, I initially lifted the alternate-site code directly from condor_meter - I tried only to adjust the parts that actually needed to be changed for this probe's implementation rather than trying to write a leaner replacement.  Apparently Edgar did the original multi-site support work in the condor probe for GRATIA-188 in 79cb619 ([here](https://github.com/opensciencegrid/gratia-probe/commit/79cb6197ee1bc40fccaf7c73c7e05ba5dd7770fb#diff-c96466053a3ee99c9d454bf9db438662de6f32ab1f82e356038345038815ad8bR929-R992)).

I am not opposed to cleaning it up though.